### DISCUSSION TITLE
GitHub Pages does not load JS script

### DISCUSSION THREAD

**Discussion by Gustavo-Ferrari** [community/community#22487] [Other Feature Feedback, Questions, & Ideas]
<p>I uploaded a simple HTML + JS project to Github Pages. The css style works fine, but it is not interacting with the user via JS.<br>
I’ve tried to move the script to below the &lt; body&gt;, leave on the &lt; HEAD&gt; with defer… everything… I could not fix it!</p>
<p>The repo is <a href="https://gustavo-ferrari.github.io/pixel-art/" class="inline-onebox" rel="noopener nofollow ugc">Projeto Pixel Art</a></p>
<p>The script link is as follows:</p>
<p>&lt; script src="/script.js" &gt;&lt; /script&gt;<br>
&lt; /body&gt;</p>
<p>Thanks a lot!</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>do the same thing like your CSS</p>
<p><code>&lt;link rel="stylesheet" href="style.css" /&gt;</code></p>
<p>remove <code>/</code></p>
<p>“/script.js” to “script.js”</p>

**Reply by Sumanth122** [reply]
my code is not working what should i do

**Comment by Gustavo-Ferrari** [comment]
<p>Thanks for the reply.<br>
Just did it and still does not work…</p>

**Comment by jfullstackdev** [comment]
<p>check your JavaScript file, is it really working ?</p>
<p>because at the Developer Tools, you will see this at the live site</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/2/e2532cb4b989f17f5e279a41dd0788d772cf3832.png" data-download-href="/uploads/short-url/wia6Lzw2ZnP4x4MTW5au4Vcchns.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181071405-40e3229d-9618-484f-ac13-4682ab4489ab.png" alt="image" data-base62-sha1="wia6Lzw2ZnP4x4MTW5au4Vcchns" width="559" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e2532cb4b989f17f5e279a41dd0788d772cf3832_2_559x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e2532cb4b989f17f5e279a41dd0788d772cf3832_2_838x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e2532cb4b989f17f5e279a41dd0788d772cf3832_2_1118x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e2532cb4b989f17f5e279a41dd0788d772cf3832_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1163×1040 89.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment]
<p>but I think it’s already working,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/6/8659ac46a28fbe42784e67e23801f298bf85c9f7.png" data-download-href="/uploads/short-url/jaw6twFsXEuoLk0fQx3Xbz9Odg3.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181071406-7a959553-767c-40b5-ab92-ef158e8f4a35.png" alt="image" data-base62-sha1="jaw6twFsXEuoLk0fQx3Xbz9Odg3" width="559" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/6/8659ac46a28fbe42784e67e23801f298bf85c9f7_2_559x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/6/8659ac46a28fbe42784e67e23801f298bf85c9f7_2_838x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/6/8659ac46a28fbe42784e67e23801f298bf85c9f7_2_1118x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/6/8659ac46a28fbe42784e67e23801f298bf85c9f7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1163×1040 51.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by Gustavo-Ferrari** [comment]
<p>I trully don’t know what happend!! I’ve removed the / and waited for the deploy… it was’t working… now, miraculously, it worked!</p>
<p>Well… i’ll not complain! :smile:</p>
<p>Thanks a lot for the help!</p>

**Comment by jfullstackdev** [comment]
<p>yup, there will be a little bit of delay for the changes to be fully in effect</p>
